src/ParquetMsg.chpl

```diff
@@ -28,10 +28,12 @@ module ParquetMsg {
   extern var ARROWINT64: c_int;
   extern var ARROWINT32: c_int;
   extern var ARROWUINT64: c_int;
+  extern var ARROWBOOLEAN: c_int;
```

Comment:
do we need an `extern var ARROWUINT32: c_int;`here? And also other places in this file? The only reason I think to ask is because `uint32` got added to the `ArrowTypes` enum

Reply:
Yes, we should probably add that, but I think it is for another PR, I'll remove the references to `uint32` that were added in this one. (I was testing some stuff and that slipped through when it shouldn't have, thanks for catching)

I've made note of this in https://github.com/Bears-R-Us/arkouda/issues/985 so we don't lose track of that.